_data/roles.yml

```diff
@@ -143,8 +150,9 @@
   description: |
     Assigning incoming issues at https://github.com/OBOFoundry/OBOFoundry.github.io, tagging them, chasing them
   responsibilities:
-    people:
-      - name: nlharris
-        orcid: 0000-0001-6315-3707
-        status: lead
-        start: 2019-01
+    - Tagging metadata related issues with 'attn\: Technical Working WG' and 'metadata'.
```

Comment:
Not sure why these particular labels, and no others, are mentioned, but ok...

Reply:
Only because they relate to the registry metadata steward! We will fill in the other ones over time!